src/matlab2tikz.m

```diff
@@ -895,7 +928,7 @@ case guitypes()
 % Get legend handle associated with current axis
 
 legendhandle = [];
-env = getEnvironment;
+env = getEnvironment();
```

Comment:
Wouldn't it make more sense to just `switch getEnvironment()` below? The values are cached anyway, so performance shouldn't be an issue.